playbooks/openshift/README.md

```diff
@@ -212,10 +242,18 @@ For initialize_ramdisk.yml to work, you will need to populate the following vari
   * openshift_cloudprovider_openstack_password
   * openshift_cloudprovider_openstack_tenant_id
   * openshift_cloudprovider_openstack_tenant_name
-  * penshift_cloudprovider_openstack_region
+  * openshift_cloudprovider_openstack_region
 
 Once you have all of this configured, running the actual installation is simple.
 
+Change the current working directory to playbooks/openshift:
+
+    $ cd ̃/git/pouta-ansible-cluster/playbooks/openshift
```

Comment:
Fixed. In my editor it was two different characters, but tilde was indeed superscripted.